qa/suites/rgw/verify/tasks/s3tests.yaml

```diff
@@ -1,5 +1,5 @@
 tasks:
 - s3tests:
     client.0:
-      force-branch: ceph-master
+      force-branch: progress-stats
```

Comment:
please make sure this piece doesn't merge. you can make a separate branch off of master with just this piece, and provide that to `teuthology-suite` using the `--suite-repo` and `--suite-branch` flags when testing this PR